src/www/pages/components/form-elements/index.njk

```diff
@@ -72,20 +104,21 @@
 
   <br><br>
   {# Input #}
-  <h2>Input</h2>
+  <h2>Text Input / Text Field</h2>
   <h3>What does it do?</h3>
-  <p>
-    Allows users to enter text.
-  </p>
+  <ul>
+    <li>Allows users to enter text.</li>
+  </ul>
   <h3>When to use it?</h3>
   <ul>
-    <li>When user needs to enter short amount of text, e.g. name, email.</li>
+    <li>When user needs to enter short amount of text, e.g. name, email. </li>
   </ul>
   <h3>When not to use it?</h3>
   <ul>
     <li>When user needs to add a lot of text that might go over multiple lines</li>
     <li>Use fixed width inputs for content that has known length, e.g. postcode.</li>
   </ul>
+  <br>
```

Comment:
You can remove this so that the spacing is consistent

Reply:
sorted!